Cargo.toml

```diff
@@ -34,4 +34,5 @@ target-lexicon = "0.13.1"
 is_ci = "1.2.0"
 sys-locale = "0.3.2"
 iana-time-zone = "0.1.61"
+tokio-test = "0.4.4"
```

Comment:
Make this a dev-dependency (or remove, since it seems unused?)

Reply:
(doc test, moved)